model_analyzer/model_manager.py

```diff
@@ -200,6 +200,10 @@ def _execute_run_configs(self):
             # Remove one run config from the list
             run_config = self._run_config_generator.next_config()
 
+            # Create model variant
+            self._create_model_variant(original_name=run_config.model_name(),
+                                       variant_config=run_config.model_config())
+
             # If this run config was already run, do not run again, just get the measurement
             measurement = self._get_measurement_if_config_duplicate(run_config)
```

Comment:
I think this requires some changes here too. This is the main place where the model config variant name is being used for duplicate detection. Could you please also add a test where Model Analyzer runs the profile command twice with completely different parameters used for sweep, then the analyze step is run and the number of rows in the table should be equal to the total of measurements run in each step?